src/navigations/root-stack.js

```diff
@@ -203,7 +226,7 @@ const AuthenticatedNavigator = () => {
         />
         <AuthenticatedStack.Screen
           name="Followings"
-          component={withSafeAreaView(withKeyboardWrapper(FollowingScreen))}
+          component={withKeyboardWrapper(FollowingScreen)}
           options={{
             headerShown: false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `withSafeAreaView` wrapper has been removed from the `FollowingScreen` component. Ensure that this does not affect the layout of the `FollowingScreen` on devices with notches or rounded corners (like iPhone X and above).



<!-- This is an auto-generated comment by OSS CodeRabbit -->